.github/workflows/go.yml

```diff
@@ -23,6 +23,16 @@ jobs:
         cd yb-voyager
         go build -v ./...
 
+    - name: Verify Build Tags in Test Files
```

Comment:
with the way that our pipelines are at the moment, if there are no tags on a test file, will they not run?